packages/loaders/src/elements/Dots.tsx

```diff
@@ -21,21 +21,13 @@ import {
 const COMPONENT_ID = 'loaders.dots';
 
 export interface IDotsProps extends React.HTMLAttributes<SVGSVGElement> {
-  /**
-   * Sets the height and width in pixels. Inherits the parent `font-size` by default
-   **/
+  /** Sets the height and width in pixels. Inherits the parent's font size by default. */
   size?: string | number;
-  /**
-   * Sets the fill color. Inherits the parent font `color` by default
-   **/
+  /** Sets the fill color. Inherits the parent's `color` by default. */
   color?: string;
-  /**
-   * Sets the speed of the loading animation in milliseconds
-   **/
+  /** Sets the length of the animation cycle in milliseconds **/
```

Comment:
omg, this is so much better. Solves a _long_ confusion.

Reply:
🎉